src/Pages/SignUp/SignUp.js

```diff
@@ -33,15 +33,15 @@ export default function SignUp() {
     setPassword(data.get("password"));
   };
 
-  const getUserToken = () => {
+  const getUserToken = async () => {
     const url = "/api/v1/auth/register";
     const data = {
       name: name,
       email: email,
       password: password,
     };
 
-    axios
+    await axios
```

Comment:
Nuriye, actually the await is not needed here, because you are using .then.  There are two ways to handle a promise, one being with await, the other being with .then.  You don't want both in the same code.

Reply:
Removed the await syntax